.github/workflows/test.yaml

```diff
@@ -115,9 +122,33 @@ jobs:
         test "$system_daily" == "$current_daily"
       shell: bash
 
+  # The required-static-checks job was introduced to maintain a consistent
+  # status check name, regardless of changes to the Go channel used for static
+  # checks. This avoids the need to update required status checks whenever the
+  # Go channel changes.
+  required-static-checks:
+    runs-on: ubuntu-latest
+    needs:
+      - static-checks
+      - branch-static-checks
+    if: always()
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          path: ./src/github.com/snapcore/snapd
+
+      - name: Confirm required static checks passed
+        uses: ./src/github.com/snapcore/snapd/.github/workflows/actions/combine-results
```

Comment:
If you let the code checkout into the current directory, then this uses simplifies:
```yaml
        uses: ./.github/workflows/actions/combine-results
```

Reply:
Thanks, done